docs/setup-cluster/k8s/resource-caps.rst

```diff
@@ -0,0 +1,100 @@
+.. _k8s-resource-caps:
+
+#####################################
+ Manage Workspace Namespace Bindings
+#####################################
+
+.. note::
+
+   Managing resource quotas in Determined requires the Determined Enterprise Edition.
+
+Determined makes it easier to manage resource quotas by handling them directly, so you don't need to
+manage them separately in Kubernetes. You can manage workspace namespace bindings and resource
+quotas using either the WebUI or CLI.
+
+*******
+ WebUI
+*******
+
+#. As an admin user, either create a new workspace or edit an existing workspace.
+
+#. In the "Namespace Bindings" section, type a name for the namespace or leave it blank. If left
+   blank, the workspace will be bound to the default namespace configured in the Master
+   Configuration YAML file.
+
+#. Toggle the "Auto Create Namespace" option on or off. When enabled, you can edit the resource
+   quota directly in Determined.
+
+#. Enter a resource quota (e.g., 5). This quota is editable only when "Auto Create Namespace" is on.
+   If the quota is set in Kubernetes, it will appear here but won't be editable.
+
+When completed, save your changes to apply the namespace bindings and resource quotas.
+
+Configure Additional Resource Managers
+======================================
+
+For each additional resource manager, configure the resource quota as needed. Each workspace in a
+cluster automatically has a default binding to their resource manager's namespace.
+
+*****
+ CLI
+*****
+
+You can use the :ref:`command-line interface (CLI) <cli-ug>` to set one workspace namespace binding
+at a time. To get help for workspace bindings, run the following command:
+
+.. code:: bash
+
+   det w bindings -h
+
+Set a Namespace Binding
+=======================
+
+To bind a workspace to a namespace for a particular cluster, use the following command:
+
+.. code:: bash
+
+   det w bindings set <workspace-id> --namespace <namespace-name>
```

Comment:
to set bindings for a particular cluster, they will have to add the `--cluster-name <cluster-name>` arg 
This field is optional in Single RM because there will only be 1 cluster

Reply:
the system needs to know the name of the cluster. This field is optional with a single resource manager scenario (because there is only one cluster)